app/utils/cyaHelpers/SummaryListRowHelper.scala

```diff
@@ -16,7 +16,7 @@
 
 package utils.cyaHelpers
 
-import models.reference.Country
+import models.reference.{Country, PackageType}
```

Comment:
Unused import?

Reply:
Yea didn't spot that when I removed the formatAsPackage function!